stats/Exemplars.md

```diff
@@ -0,0 +1,48 @@
+Exemplars
+===
+
+Exemplars are example data points for aggregated data. In particular, for distribution-type
+metrics, exemplars are points associated with each bucket in the distribution giving an
+example of what was aggregated into the bucket.
+
+Exemplars consist of a value that was recorded, an exact timestamp of when that value
+was recorded, and a set of attachments.
+
+Attachments
+---
+
+Exemplars my have zero or more attachments. Attachments are key-value pairs that
+describe the context in which the exemplar was recored.
+
+Well-known attachment keys
+---
+
+We define the following types of attachments:
+
+1. `TraceID`: lower-case base16 encoded trace ID
+2. `SpanID`:  lower-case base16 encoded span ID
+3. `TraceOptions`: lower-case base16 encoded trace options
```

Comment:
this is kind of irrelevant as you asking to sample it anyway. Or this document tells to override the sampling flag for downstream services as well?

Reply:
Yeah agreed, I will remove it for now.

Reply:
and also make a note whether this document asks for sampling individual span in or everything downstream as well